pkg/kubelet/network/cni/cni.go

```diff
@@ -53,14 +59,18 @@ type cniNetwork struct {
 
 func probeNetworkPluginsWithVendorCNIDirPrefix(pluginDir, vendorCNIDirPrefix string) []network.NetworkPlugin {
 	configList := make([]network.NetworkPlugin, 0)
-	network, err := getDefaultCNINetwork(pluginDir, vendorCNIDirPrefix)
-	if err != nil {
-		return configList
-	}
-	return append(configList, &cniNetworkPlugin{
-		defaultNetwork: network,
+	plugin := &cniNetworkPlugin{
+		defaultNetwork: nil,
```

Comment:
Ok, I no longer check initialization in the status. That should fix this issue I believe. The longterm fix is to decouple pod network and node network readiness in the runtime status check.